src/Network/Network/FirewallPolicy/PolicySettings/AzureApplicationGatewayFirewallPolicySetting.cs

```diff
@@ -43,6 +51,10 @@ public class AzureApplicationGatewayFirewallPolicySetting : NetworkBaseCmdlet
         [ValidateNotNullOrEmpty]
         public int MaxRequestBodySizeInKb { get; set; }
 
+        [Parameter(Mandatory = false, HelpMessage = "Whether allow WAF to enforce file upload limits.")]
+        [ValidateNotNullOrEmpty]
+        public bool FileUploadEnforcement { get; set; }
```

Comment:
Should we rather change to EnableFileUploadEnforcement?

Reply:
same as above

Reply:
We should have changed it there :( I wonder why swagger team didn't raise it